crates/routes/src/feeds.rs

```diff
@@ -124,12 +124,15 @@ async fn get_feed_data(
 }
 
 async fn get_feed(
-  web::Path((req_type, param)): web::Path<(String, String)>,
+  req: HttpRequest,
   info: web::Query<Params>,
   context: web::Data<LemmyContext>,
 ) -> Result<HttpResponse, Error> {
   let sort_type = get_sort_type(info).map_err(ErrorBadRequest)?;
 
+  let req_type: String = req.match_info().get("type").unwrap_or("none").parse()?;
+  let param: String = req.match_info().get("name").unwrap_or("none").parse()?;
```

Comment:
Hmm that seems to be the same as before. Why did you remove the web::Path at all, did they remove that?

Reply:
Yep they removed that tuple and made it private.

Reply:
Damn